charon/src/cli_options.rs

```diff
@@ -110,27 +123,39 @@ performs: `y := (x as E2).1`). Producing a better reconstruction is non-trivial.
 Report the errors as warnings
 "
     )]
+    #[serde(default)]
     pub errors_as_warnings: bool,
+    #[structopt(
+        long = "no-serialize-llbc",
+        help = "
+Don't serialize the final LLBC to a file.
+"
+    )]
+    #[serde(default)]
+    pub no_serialize_llbc: bool,
```

Comment:
It would be better to name this `no-serialize` as we can serialize ULLBC as well?